cluster/OWNERS

```diff
@@ -8,6 +8,7 @@ reviewers:
   - mikedanese
   - spiffxp
 approvers:
+  - sig-architecture-approvers
```

Comment:
this is the only one I'm unsure of... is arch the right sig for this folder?

Reply:
cloud provider or testing fits for the actual cluster bits (formerly cluster lifecycle...).

cluster/kubemark is scalability, cluster/images/ should probably be moved and is a mix of testing / release / ...

Reply:
I do not have a strong bias one way or the other.  I am happy to have approve here, I have history on kube-up, most of this content at this level (not in a subtree) is under maintenance.  this change basically gives me and john new privileges.  dims appear to already have the privilege.

Reply:
@liggitt used to be cluster-lifecycle a while ago. we need to split things up like Ben said and move bits and pieces off to other sigs. having additional hands to review/approve would be useful

Reply:
> we need to split things up like Ben said and move bits and pieces off to other sigs

that's fine, but sig-arch seems strange for this one. do we expect all future members of sig-architecture-approvers to have context on this dir?

Reply:
agree it's strange.

Reply:
if you want to drop this line, the rest of the PR looks helpful, we can iterate on this bit and hopefully target it where it makes more sense

Reply:
this still appears out of place to me.

Reply:
Done!